README.md

```diff
@@ -1,410 +1,61 @@
-| `spot_price`        | float with 5 decimals at most            | -1                                                                        | Sets the maximum price that you are willing to pay by the hour. If not specified, the current spot bidding pricing will be used                                                                                                                                                                  |
-| `name`              |                                          | iterative\_{UID}                                                          | Sets the instance name and related resources based on that name. In Azure, groups everything under a resource group with that name.                                                                                                                                                              |
-| `instance_hdd_size` |                                          | 10                                                                        | Sets the instance hard disk size in GB                                                                                                                                                                                                                                                           |
-| `instance_type`     | `m`, `l`, `xl`                           | `m`                                                                       | Sets the instance CPU size. You can also specify vendor specific machines in AWS i.e. `t2.micro`. [See equivalences](#Supported-vendors) table below.                                                                                                                                            |
-| `instance_gpu`      | ``, `tesla`, `k80`                       | ``                                                                        | Selects the desired GPU for supported `instance_types`.                                                                                                                                                                                                                                          |
-| `ssh_private`       |                                          |                                                                           | An SSH private key in PEM format. If not provided, one private and public key wll be automatically generated and returned in `terraform.tfstate`                                                                                                                                                 |
-
-### Machine
+- Rapidly move local machine learning experiments to a cloud infrustructure
```

Comment:
```suggestion
- Rapidly move local machine learning experiments to cloud infrastructure
```

Reply:
"move to a cloud infrastructure" feels correct to me as there can be different kinds of it.

Reply:
```suggestion
- Rapidly move local machine learning experiments to a cloud infrastructure
```

Reply:
👌🏼  Left the controversial “a” in place. 😄  Still, it sounds a bit artificial. Pinging @casperdcl and his opinions... 🪔 🧞‍♂️